virtualenv.py

```diff
@@ -1391,7 +1395,7 @@ def install_python(home_dir, lib_dir, inc_dir, bin_dir, site_packages, clear, sy
         else:
             raise e
 
-    proc_stdout = proc_stdout.strip().decode("utf-8")
+    proc_stdout = proc_stdout.strip().decode(sys.getfilesystemencoding())
```

Comment:
This patch is no longer necessary after switching back to "always-utf8" protocol.